fischkapp/src/App.tsx

```diff
@@ -4,12 +4,25 @@ import { theme } from "./components/styles/theme/theme";
 import { Wrapper } from "./components/styles/Wrapper";
 import { Header } from "./components/Header";
 import { GlobalStyle } from "./components/styles/GlobalStyles";
+import { FrontCard } from "./components/FrontCard";
+import { BackCard } from "./components/BackCard";
+import { useState } from "react";
+
 function App() {
+  const [cardStatus, setCardStatus] = useState<boolean>(true);
```

Comment:
**nitpick:** Im not sure what `cardStatus` means as it's of boolean type - I think we should improve better naming here